ENGAGEHF/Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,75 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+//  DashboardView.swift
+//  ENGAGEHF
+//
+//  Created by Nick Riedman on 4/3/24.
+//
+
+import SpeziAccount
+import SpeziMockWebService
+import SwiftUI
+
+struct Greeting: View {
+    @State private var dateString: String?
+    
+    var body: some View {
```

Comment:
Different elements of a type (e.g. here stored properties and computed properties) should be separated by two empty lines:
```suggestion
    @State private var dateString: String?
    
    
    var body: some View {
```
Within each group of types I would recommend to use no empty lines or a single empty line if you need to logically group elements.